packages/visx-annotation/src/components/LineSubject.tsx

```diff
@@ -29,9 +29,8 @@ export default function LineSubject({
   min,
   max,
   stroke = '#222',
-  strokeWidth = 2,
   ...restProps
-}: LineSubjectProps & Omit<React.SVGProps<SVGLineElement>, keyof LineSubjectProps>) {
+}: LineSubjectProps) {
```

Comment:
can you send a link that reproduced the TS error you are seeing? It works okay for me in the `/annotation` [codesandbox](https://codesandbox.io/s/dazzling-jang-zxpkl?file=/Example.tsx)

![image](https://user-images.githubusercontent.com/4496521/103814580-c20bfc00-5016-11eb-8bf3-fd50e70b8c6e.png)

For other packages/components we have preferred to set the native SVG props on the component as a union with its own exported prop types, rather than on the exported prop types themselves to keep them separate and simplify re-use of the export where relevant.

Reply:
<img width="805" alt="Screenshot 2021-01-06 at 22 09 27" src="https://user-images.githubusercontent.com/2109932/103820048-e0501700-506b-11eb-99c6-24db557d116f.png">

I actually saw it on `AnnotationLineSubject`. I admit I didn't research this thoroughly enough and just assumed it propagated from `LineSubject`

Reply:
can be reproduced in the `xychart` codesandbox https://codesandbox.io/s/github/airbnb/visx/tree/master/packages/visx-demo/src/sandboxes/visx-xychart?file=/Example.tsx:7459-7501

Reply:
@williaster Update the PR to update `AnnotationLineSubject` instead